docs/guide/app-block-type-select.md

```diff
@@ -20,4 +20,17 @@ Using select with string values:
     ['value' => 'john', 'label' => 'John Doe'],
     ['value' => 'jane', 'label' => 'Jane Doe'],
 ]],
-```
\ No newline at end of file
+```
+
+The <a href="http://luya.io/api/luya-cms-helpers-BlockHelper">Blockhelper function provides you an optimized markup for the select options array.
```

Comment:
Please take a look at how to create internal links => https://luya.io/guide/luya-guideline

Your link would even have missed the closing tag